open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/json/BoundingBox.java

```diff
@@ -77,4 +75,34 @@ public void setTop_left(Coordinate topLeft)
     {
         myTopLeft = topLeft;
     }
+
+    /**
+     * Get the smallest bounding box which contains all of the coordinates.
+     *
+     * @param coordinates The coordinates which must be contained in the box.
+     * @return The smallest bounding box which contains all of the coordinates.
+     */
+    static GeographicBoundingBox getMinimumBoundingBoxLLA(com.vividsolutions.jts.geom.Coordinate[] coordinates)
+    {
+        double minLat = Double.MAX_VALUE;
+        double maxLat = -Double.MAX_VALUE;
+
+        double minLon = Double.MAX_VALUE;
+        double maxLon = -Double.MAX_VALUE;
+
+        for (com.vividsolutions.jts.geom.Coordinate coord : coordinates)
+        {
+            double latD = coord.y;
+            minLat = Math.min(minLat, latD);
+            maxLat = Math.max(maxLat, latD);
+
+            double lonD = coord.x;
+            minLon = Math.min(minLon, lonD);
+            maxLon = Math.max(maxLon, lonD);
+        }
+
+        GeographicBoundingBox bbox = new GeographicBoundingBox(LatLonAlt.createFromDegrees(minLat, minLon),

```

Comment:
Can you just return `new GeographicBoundingBox(LatLonAlt.createFromDegrees(minLat, minLon),` instead of assigning it a to a variable

Reply:
i mean return the whole new expression i forgot to copy rest of code

Reply:
Done